students/jarosz_natalia/lesson_09/extending_the_multiclipboard.py

```diff
@@ -15,8 +15,13 @@ def main():
     # TODO: Save clipboard content.
     if len(sys.argv) == 3 and sys.argv[1].lower() == 'save':
             mcb_shelf[sys.argv[2]] = pyperclip.paste()
+    # Extend the multiclipboard program in this so that it has a delete_all
+    # command line argument that will delete all keyword from the shelf.
+    elif len(sys.argv) == 2 and sys.argv[1].lower() == 'delete_all':
+        print("test1")
```

Comment:
This print shouldn't be here 😄 No worries task accepted, just remember to avoid such not needed prints. in the lesson 12, you will learn some technics of debugging 😄